docs/src/main/sphinx/release/release-406.md

```diff
@@ -81,6 +85,8 @@
 
 * Add support for changing column types. ({issue}`15515`)
 * Add [support for the JDBC catalog](iceberg-jdbc-catalog). ({issue}`9968`)
+* Add support for changing fields in the `row` type. ({issue}`15808`)
```

Comment:
> Add support for changing fields in columns of `row` type.

BTW, the original PR has this comment for the RN: "with the following suggested text if this PR isn't merged in 406". Does that mean that if we merge it in 406 we don't need RN? Why is that @ebyhr ?

Reply:
Make no sense to me. I would just leave it in and ship the release. If really necessary we can take out after .. although I dont understand why that would be the case..

Reply:
This entry is unnecessary becasue it's already covered by another entry:
```
Add support for changing column types. ({issue}`15515`)
```